elife/php-long-running-processes.sls

```diff
@@ -0,0 +1,47 @@
+# only supports 16.04!
+
+{% if pillar.elife.php.processes.enabled %}
+{% for name, configuration in pillar.elife.php.processes.configuration.items() %}
+{% set hyphenized = name | replace('_', '-') %}
+{% set service_name = salt['elife.project_name']() + '-' + hyphenized %}
+php-long-running-process-service-{{ hyphenized }}:
+    file.managed:
+        - name: /lib/systemd/system/{{ service_name }}@.service
+        - source: salt://elife/config/lib-systemd-system-php-service.service
+        - template: jinja
+        - context:
+            name: {{ hyphenized }}
+            folder: {{ configuration.folder }}
+            command: {{ configuration.command }}
+        - require:
+            - php
```

Comment:
Is this the only thing here that makes it PHP-specific?

Reply:
Also we won't get away with https://github.com/elifesciences/builder-base-formula/pull/87/files#diff-568b5141ae75a6b187f83bb22ee62d04R14 for Python as it requires New Relic magic around it:
https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/config/lib-systemd-system-uwsgi-profiles.service#L15
possibly `KillSignal` and `Type` need configurability too